src/spatial/collision.js

```diff
@@ -350,38 +342,25 @@ Crafty.c("Collision", {
     },
 
 
-    /**@
-     * #._createCollisionHandler
-     * @comp Collision
-     * @sign public void .checkHits(String component, Object collisionData)
-     * @param component - The name of the component for which this handler
-     * checks for collisions.
-     * @param collisionData - Collision data object used to track collisions with
-     * the specified component.
-     *
-     * This is a helper method for creating collisions handlers set up by
-     * `.checkHits(...)`. Do not call this directly.
-     *
-     * @see .checkHits
-     */
+    // This is a helper method for creating collisions handlers set up by checkHits method.
```

Comment:
I decided to remove all jsdoc after your [comment](https://github.com/craftyjs/Crafty/pull/949/files#r35801524) just to make it clear to other people.